doc/pachctl/pachctl.rst

```diff
@@ -1,51 +0,0 @@
-Pachctl Command Line Tool
```

Comment:
Sure. `make doc` seems to want to remove it (though I admin don't actually know what `make doc` is doing), but I got around that by running `make doc` and then checking out this file from `HEAD~1`